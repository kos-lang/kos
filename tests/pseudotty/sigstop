print(123)\r

456

# Ctrl-Z generates SIGTSTP
\x1A

+1\r

\x10 # Up
\r

# Exit
\x04
