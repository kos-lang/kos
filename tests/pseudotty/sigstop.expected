R:Kos 0.1 interactive interpreter\r
R:\e[9999C\e[6n\r>
R: \e[0Kprint(123)\r
R:123\r
R:\e[9999C\e[6n\r>
R: \e[0K456\r> 456\e[0K+1\r
R:457\r
R:\e[9999C\e[6n\r>
R: \e[0K\r> 456+1\e[0K\r
R:457\r
R:\e[9999C\e[6n\r>
R: \e[0K
EXIT:0
